roles/sap_general_preconfigure/tasks/sapnote/2369910.yml

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+- name: Configure - Display SAP note number 2369910 and its version
+  ansible.builtin.debug:
+    msg: "SAP note {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2369910$') | first).number }}
+          (version {{ (__sap_general_preconfigure_sapnotes_versions | selectattr('number', 'match', '^2369910$') | first).version }}): SAP Software on Linux: General Information"
+  tags:
+    - always
+
+- name: Check locales
+  when: sap_general_preconfigure_config_all | d(true) or sap_general_preconfigure_2369910 | d(false)
+  tags:
+    - sap_general_preconfigure_2369910
+    - sap_general_preconfigure_configure_locale
+  block:
+    - name: Get list of installed locales
+      ansible.builtin.command: locale -a
+      changed_when: false
+      register: __sap_general_preconfigure_locales_installed
+
+    - name: Assert that an English locale is installed
+      ansible.builtin.assert:
+        that: __sap_general_preconfigure_locales_installed.stdout_lines | select('match', '^en_') | list | length > 0
```

Comment:
Maybe this one is better, as it looks for ' en_[A-Z]*' AND Unicode:
```
__sap_general_preconfigure_locales_installed.stdout_lines | regex_findall('en_[A-Z]*[.](utf8|UTF-8)') | length > 0
```

Reply:
Update: As we only have to take into account operating systems on which the command `localectl` is available (RHEL 7+ and SLES15+), we do not have to check if an English locale is installed. Starting with RHEL 8 and SLES15, the `localectl set-locale` command is checking if a locale is also installed. For RHEL 7 managed nodes, we can cover this topic in the documentation.

So we can remove the tasks `Get list of installed locales` and `Assert that an English locale is installed`.